docs/source/Advanced-Usage.md

```diff
@@ -86,6 +86,21 @@ List of some of the analyzers with optional configuration:
     * `mxcheck` (default False): find suspicious mail servers and flag them with SPYING-MX string.
     * `tld` (default False): check for domains with different TLDs by supplying a dictionary file.
     * `tld_dict` (default abused_tlds.dict): dictionary to use with tld argument. (common_tlds.dict/abused_tlds.dict)
+* `ZoomEye`:
+  * Provide the `search_type`, `query`, `page`, `facets` values through `runtime_configuration`. Note that `search_type` is one of the below choices.
```

Comment:
2 things,

1. The parameters should be explained in a bullet list style manner. Saying specify x,y,z values through runtime_configuration is already established above. So just convert this into this format,

```suggestion
* ZoomEye:
    - `search_type`: ...
    - `facets`: ....
```

The example is also not required. Just provide an example inside the bullet list against each parameter.

2. The markdown is breaking somewhere when rendered, fix that

Reply:
Shall I squash this commit made by you?